packages/account/src/Stores/init-store.js

```diff
@@ -0,0 +1,16 @@
+import { configure } from 'mobx';
+import RootStore from 'Stores';
+import { setWebsocket } from 'Services/ws-methods';
+
+configure({ enforceActions: 'observed' });
+
+const initStore = (core_store, websocket) => {
```

Comment:
Is this needed in accounts? How come the trade store is initialised here?

Reply:
Per discussion on slack, I mark this as resolved.